test/reader.test.js

```diff
@@ -41,17 +41,6 @@ test('should successfully bundle 2 feeds', async () => {
     expect(clean(prettier.format(content))).toMatchSnapshot();
 });
 
-test('should successfully bundle 2 feeds using automatic temp directory', async () => {
```

Comment:
remove the code for automatic temp as well, and just require a passed in directory?

Reply:
It would be nice not to have to change the api or pass in what feels like an internal detail no?

Reply:
We're unable to test it though, can break at any time.

API wise it's nicer though, so maybe we should investigate the CI failure?

Reply:
yea, probably should eh.